clients/admin-ui/src/features/datastore-connections/constants.ts

```diff
@@ -19,44 +19,6 @@ export enum DisabledStatus {
   DISABLED = "disabled",
 }
 
-export enum SaasType {
```

Comment:
Removing `SaaSType` enum, this needs to be dynamic and driven by the registered SaaS connectors